infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerExposer.java

```diff
@@ -256,22 +256,26 @@ private void splitServersAndPortsByExposureType(
   private void exposeNonSecureServers(
       Map<String, ServerConfig> internalServers,
       Map<String, ServerConfig> externalServers,
-      Map<String, ServicePort> unsecuredPorts)
+      Map<String, ServicePort> unsecuredPorts,
+      Map<String, ServicePort> securedPorts)
```

Comment:
handling `securedPorts` in `exposeNonSecureServers` method is confusing. Can you please explain what exactly are you doing here and, most importantly, what consequences this changes will have?

Reply:
Youre' right, naming is confusing and didn't refrect reality. I'll try to rearrange this method.